pynicotine/networkfilter.py

```diff
@@ -511,9 +511,9 @@ def check_user(self, username, ip_address=None):
         user_data = core.userlist.buddies.get(username)
 
         if user_data:
-            if config.sections["transfers"]["buddysharestrustedonly"] and not user_data.is_trusted:
-                # Only trusted buddies allowed, and user isn't trusted
-                return 1, ""
+            if user_data.is_trusted:
+                # For sending trusted buddy shares
+                return 3, ""
```

Comment:
Consider returning normalized strings from this function instead of numeric integers.